signature/src/schnorr.rs

```diff
@@ -101,21 +101,15 @@ where
 // =====================================================
 // Signing key
 // =====================================================
-#[tagged(tag::SCHNORR_SIGNING_KEY)]
-#[derive(
-    Clone,
-    Hash,
-    Default,
-    Zeroize,
-    Eq,
-    PartialEq,
-    CanonicalSerialize,
-    CanonicalDeserialize,
-    Derivative,
-)]
-#[derivative(Debug)]
+#[derive(Clone, Hash, Default, Zeroize, Eq, PartialEq)]
 /// Signing key for Schnorr signature.
-pub struct SignKey<F: PrimeField>(#[derivative(Debug = "ignore")] pub(crate) F);
+pub struct SignKey<F: PrimeField>(pub(crate) F);
+
+impl<F: PrimeField> core::fmt::Debug for SignKey<F> {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("<private signing key>").finish()
+    }
+}
```

Comment:
1. add comment explain that `Debug` even though didn't print anything, will makes outer struct easily derive Debug.
2. previously we had `tag::SCHNORR_SIGNING_KEY`, we should also prefix our debug message with that domain-separate tag. (don't have to be exactly tagged-base64 style of tag, but some kind of prefix indicating the type of key this is)

Reply:
How about just `BLSSignKey`, `SchnorrSignKey`, etc.

Reply:
Sure, that works.

Reply:
done in [5f85f10](https://github.com/EspressoSystems/jellyfish/pull/696/commits/5f85f1048212d738738a5884784269f1aea54d71)